recipes/hdf5/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: hdf5
+  version: 1.8.16
+
+source:
+  fn: hdf5-1.8.16.tar.gz
+  url: http://www.hdfgroup.org/ftp/HDF5/current/src/hdf5-1.8.16.tar.gz
+  md5: b8ed9a36ae142317f88b0c7ef4b9c618
+
+
+build:
+  features:
+    - vc9     # [win and py27]
+    - vc10    # [win and py34]
+    - vc14    # [win and py35]
```

Comment:
Ah, ok, thanks for setting me straight.

I had a nice chat with @ukoethe and @stuarteberg earlier. There are some proposals that @ukoethe has for how we may help address these sort of things in `conda build` and more generally ABI and API compatibility. Also, I think it would be great if we can we work with @ukoethe to incorporate his work on building Python with a looser VS constraint as a proposal at conda-forge.